src/derive/curve.rs

```diff
@@ -49,12 +49,19 @@ macro_rules! endo {
     };
 }
 
+// These masks work for 0, 1, and 2 spare bits.
+// The only curve that does not fit these masks is Bls12-381.
+pub(crate) const NEG_Y_MASK: u8 = 0b1000_0000;
+pub(crate) const NEG_Y_SHIFT: u8 = 7;
```

Comment:
Hmm, I think we can give more general name for this `NEG_Y_*`, like `SIGN_*`? 
What do you think? @davidnevadoc